model_clone/mixins/clone.py

```diff
@@ -321,12 +321,16 @@ def _create_copy_of_instance(instance, using=None, force=False, sub_clone=False)
         new_instance = cls()
 
         for f in fields:
-            value = f.get_default() or getattr(instance, f.attname, None)
-
+            # Use default value for special fields like timestamp, uuid
```

Comment:
We can remove the comment and `isinstance(f, (models.DateTimeField, models.DateField)):`

Since this condition would be true for all non editable fields.

```python
if not f.editable:
    f.pre_save(new_instance, add=True)
    continue
```

Reply:
Good catch, will update it.